boris-service/main/boris-build.hs

```diff
@@ -11,7 +11,6 @@ import qualified Boris.Service.Discover as Discover
 import           Data.Default (def)
 import           Data.String (String)
 
-import           Mismi (renderRegionError, discoverAWSEnv)
```

Comment:
This was just an experiment of trying to implement something solo, I didn't really mean to push it to master, but it can stay and I will polish later if it isn't in your way.